pug-practice/Dockerfile

```
@@ -1,4 +1,6 @@
-FROM timche/csgo:sourcemod
+FROM timche/csgo
```

This image must stay based on `timche/csgo:sourcemod`, otherwise the `server_sourcemod.sh` script is missing.
```suggestion
FROM timche/csgo:sourcemod
```